app/models/tutorial.rb

```diff
@@ -2,8 +2,8 @@ class Tutorial < ApplicationRecord
   has_many :videos, -> { order(position: :asc) }, dependent: :destroy
   acts_as_taggable_on :tags, :tag_list
   accepts_nested_attributes_for :videos
-
-  validates_format_of :thumbnail, with: %r{ (\Ahttps:\/\/i\.ytimg\.com\/vi\/([a-zA-Z0-9_-]*)\/hqdefault\.jpg\Z) }, on: :create
```

Comment:
Hey we should keep the regex in here, as that's how I read the spec on #1 : 
`And I fill in 'thumbnail' with a valid YouTube thumbnail`